apprentice_teacher.rb

```diff
@@ -1,23 +1,17 @@
-class ApprenticeTeacher
+require_relative 'people'
+require_relative 'teacher'
+class ApprenticeTeacher < People
```

Comment:
This says that an apprentice teacher is a people.  Always name your class a name that makes sense for a single instance of the class, not all instances (so `Person`, not `People`)